cache_images/fedora_packaging.sh

```diff
@@ -159,36 +152,25 @@ DOWNLOAD_PACKAGES=(\
     parallel
 )
 
-echo "Installing general build/test dependencies for Fedora '$OS_RELEASE_VER'"
-$BIGTO ooe.sh $SUDO dnf install -y ${INSTALL_PACKAGES[@]}
-
-# AD-HOC CODE FOR SPECIAL-CASE SITUATIONS!
-# On 2020-07-23 we needed this code to upgrade crun on f31, a build
-# that is not yet in stable. Since CI:IMG PRs are a two-step process,
-# the key part is that we UN-COMMENT-THIS-OUT during the first step,
-# then re-comment it on the second (once we have the built images).
-# That way this will be dead code in future CI:IMG PRs but will
-# serve as an example for anyone in a similar future situation.
-#   $BIGTO ooe.sh $SUDO dnf --enablerepo=updates-testing -y upgrade crun
+echo "Installing general build/test dependencies"
+bigto ooe.sh $SUDO dnf install -y "${INSTALL_PACKAGES[@]}"
 
 [[ ${#REMOVE_PACKAGES[@]} -eq 0 ]] || \
-    $LILTO ooe.sh $SUDO dnf erase -y "${REMOVE_PACKAGES[@]}"
+    lilto ooe.sh $SUDO dnf erase -y "${REMOVE_PACKAGES[@]}"
 
 if [[ ${#DOWNLOAD_PACKAGES[@]} -gt 0 ]]; then
     echo "Downloading packages for optional installation at runtime, as needed."
     # Required for cri-o
-    ooe.sh $SUDO dnf -y module enable cri-o:$(get_kubernetes_version)
+    ooe.sh ooe.sh $SUDO dnf -y module enable cri-o:$(get_kubernetes_version)
```

Comment:
Do you intentionally want two `ooe.sh`s? If so, can you document why? It's not really clear to me what this does.

Reply:
woopsies...that was a search-replace goof.